packages/shared-kernel/package.json

```diff
@@ -40,23 +40,23 @@
   "dependencies": {
     "decimal.js": "^10.4.3",
     "lodash": "^4.17.21",
-    "tslib": "^2.4.1",
+    "tslib": "^2.5.0",
     "uuid": "^9.0.0"
   },
   "devDependencies": {
     "@babel/preset-env": "^7.20.2",
     "@babel/preset-typescript": "^7.18.6",
     "@lens-protocol/eslint-config": "workspace:*",
     "@lens-protocol/prettier-config": "workspace:*",
-    "@types/jest": "^29.2.3",
+    "@types/jest": "29.2.3",
```

Comment:
Same here, is the `^` missing intentional?

Reply:
yes, the latest version of types/jest is missing the correct type resolution for `expect.any()` 
I don't know why